src/auth/screens/login.screen.js

```diff
@@ -39,6 +47,32 @@ const styles = StyleSheet.create({
     ...fonts.fontPrimaryBold,
     fontSize: normalize(12),
   },
+  browserDismiss: {
+    flex: 1,
+  },
+  browserLoader: {
+    flex: 1,
+    position: 'absolute',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '100%',
+    width: '100%',
+    textAlign: 'center',
+    backgroundColor: '#1f2327',
```

Comment:
Let's put `#1f2327` in the `colors` file. Can call it whatever you think sounds reasonable (`githubDark` or something if you like).